docs/source/index.rst

```diff
@@ -6,19 +6,37 @@
 Welcome to the GEMD-Python Documentation!
 ===================================================
 
-``gemd`` is the Python implementation of the Citrine data model, the full documentation of which
-can be found `here <https://citrineinformatics.github.io/gemd-docs/>`_.
+This site documents the Python implementation of the Graphical Expression of Materials Data (GEMD) model.
+GEMD is an open source data format developed by `Citrine Informatics <https://citrine.io/>`_ for representing
+data in materials in a `FAIR <https://www.go-fair.org/fair-principles/>`_ and transformable manner.
+Documentation of the underlying data model can be found `here <https://citrineinformatics.github.io/gemd-docs/>`_.
 
 To learn about the details of specific classes, please see the module index.
 
+Installation
+------------
+
+The latest release can be installed via `pip`:
+
+.. code::
+
+  pip install gemd
+
+or a specific version can be installed, for example:
+
+.. code::
+
+  pip install gemd==1.0.0
```

Comment:
Since I imagine many users will just copy this line, let's make it the current version. Of course, it's impractical to update it on every release, but we might as well while we're here.

Reply:
Updated.  It's unfortunate that you can't substitute in a code block.